salt/modules/file.py

```diff
@@ -4868,6 +4997,26 @@ def check_file_meta(
 
     contents
         File contents
+
+    seuser
+        selinux user attribute
+
+        .. versionadded:: Neon
+
+    serole
+        selinux role attribute
+
+        .. versionadded:: Neon
+
+    setype
+        selinux type attribute
+
+        .. versionadded:: Neon
+
+    serange
+        selinux range attribute
+
+        .. versionadded:: Neon
```

Comment:
These are OK since they denote individual arguments :+1: 

It's just the ones that apply to the entire function that need to be at the top.